src/java.base/share/classes/java/time/format/Parsed.java

```diff
@@ -446,6 +468,36 @@ private void resolveTimeFields() {
             updateCheckConflict(MINUTE_OF_DAY, HOUR_OF_DAY, mod / 60);
             updateCheckConflict(MINUTE_OF_DAY, MINUTE_OF_HOUR, mod % 60);
         }
+        if (dayPeriod != null) {
+            if (fieldValues.containsKey(HOUR_OF_DAY)) {
```

Comment:
MINUTE_OF_HOUR without HOUR_OF_DAY/HOUR_OF_AMPM may not make any sense, so it is only validated in updateCheckDayPeriodConflict.

Reply:
But can you get a CLDR rule that says "at night" before 05:30 and "In the morning" from 05:30 onwards? If you can then I don't think it is handled, because HOUR_OF_DAY and MINUTE_OF_HOUR are not used together when checking against DayPeriod.

Reply:
CLDR allows rules that involve MINUTE_OF_HOUR. The validation I meant was within the `updateCheckDayPeriodConflict`:
```
            long hod = fieldValues.get(HOUR_OF_DAY);
            long moh = fieldValues.containsKey(MINUTE_OF_HOUR) ? fieldValues.get(MINUTE_OF_HOUR) : 0;
            long mod =  hod * 60 + moh;
            if (!dayPeriod.includes(mod)) {
                throw new DateTimeException("Conflict found: Resolved time %02d:%02d".formatted(hod, moh) +
                        " conflicts with " + dayPeriod);
            }
```
which checks both hod/moh. moh is assumed zero if `MINUTE_OF_HOUR` does not exist. Are you suggesting `HOUR_OF_DAY` should also be assumed zero if it does not exist?